Resources/config/dbal.xml

```diff
@@ -35,7 +35,7 @@
         </service>
 
         <service id="data_collector.doctrine" class="%doctrine.data_collector.class%" public="false">
-            <tag name="data_collector" template="@Doctrine/Collector/db.html.twig" id="db" />
+            <tag name="data_collector" template="@Doctrine/Collector/db.html.twig" id="db" priority="265" />
```

Comment:
Yes. This is how the profiler menu looks:

![profiler_menu](https://cloud.githubusercontent.com/assets/73419/10195881/3795f428-6790-11e5-93a7-e631c068ba65.png)